app/validation/validate_schema_text.py

```diff
@@ -0,0 +1,64 @@
+import re
+from jsonpointer import resolve_pointer
+from eq_translations.utils import find_pointers_to
+from eq_translations.survey_schema import SurveySchema
+
+
+def get_pointer_keys():
+    pointer_keys = [
+        'show_guidance',
+        'hide_guidance',
+        'description',
+        'legal_basis',
+        'playback',
+        'item_title',
+        'label',
+        'add_link_text',
+        'empty_list_text',
+        'text',
+        'title',
+    ]
+
+    return pointer_keys
+
+
+def get_pointer_contents(schema, keys):
+    contents = []
+    for key in keys:
+        schema_elements = find_pointers_to(schema, key)
+        for element in schema_elements:
+            pointer_contents = resolve_pointer(schema, element)
+            contents.append(pointer_contents)
+
+    return contents
+
+
+def get_list_pointer_contents(schema, schema_elements):
+    contents = []
+    for element in schema_elements:
+        pointer_contents = resolve_pointer(schema, element)
+        contents.append(pointer_contents)
+    return contents
+
+
+def search_dumb_quotes(text):
+    return re.search(r"'", text)
```

Comment:
If the other changes are made and this is only called from one place, I don't think this should be a separate method.

Reply:
Removed the method